src/main/java/core/basesyntax/model/enums/Operation.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.model.enums;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+public enum Operation {
+    BALANCE("b"), SUPPLY("s"), PURCHASE("p"), RETURN("r");
+
+    private final String code;
+
+    Operation(String code) {
+        this.code = code;
+    }
+
+    public static Operation fromCode(String code) {
+        String exceptionMessage = "Operation with code = [" + code + "] not found";
```

Comment:
let's use this exception message directly without creating an additional variable. Fix same issue in all places